src/components/grid-aware/OneParagraphBlock/OneParagraphBlock.module.css

```diff
@@ -0,0 +1,125 @@
+/* Outer grid */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedBackground {
+  background: var(--color-black);
+  grid-column: 2 / span 2;
+  grid-row: 1;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+  }
+
+  .bleedBackground {
+    grid-column: 1 / span 3;
+  }
+}
+
+
+/* Main grid */
+
+.gridParent {
+  color: var(--color-white);
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-column-start: 2;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  padding: 140px 0 93px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding: 50px 0 80px;
+  }
+}
+
+
+/* Main grid areas */
+
+.gridAreaLeft {
+  grid-column: 2 / span 4;
+}
+
+.gridAreaRight {
+  grid-column: 7 / span 5;
+}
+
+
+/* Title block */
+
+.title {
+  font: var(--font-headline);
+}
+
+
+/* Jobs block */
+
+.jobsWrapper {
+  margin-top: 0;
+}
+
+@media (--tablet-and-down) {
+  .jobsWrapper {
+    margin-top: 40px;
+  }
+}
+
+.jobs *:first-child {
```

Comment:
OK, I see what you're doing here now. I think this is one of the few places where I think using tag selectors is appropriate, since in the future, we'll expect the actual copy to come from non-technical users through the CMS, where they'll likely be editing through a rich-text interface. It'll eventually to us in the form of [Rich Text](https://user-guides.prismic.io/en/articles/383762-rich-text) objects from Prismic, which boils down to simple HTML elements.

For now, how about you mimic this by using heading tags (`<h1>`, `<h2>`, ...) and paragraph tags (`<p>`) and styling them based on the tag name? This will be more robust than hoping that we will only have one paragraph per heading.